packages/spec/integration/graphql/index.js

```diff
@@ -83,14 +35,4 @@ const App = () => (
   </>
 );
 
-export default render(<App />, {
-  graphql: {
-    resolvers: {
-      Query: {
-        showCommits() {
-          return true;
-        },
-      },
-    },
-  },
-});
+export default render(<App />);
```

Comment:
Do we want to keep the render-options `resolvers`? They're not checked in the other graphql-related tests. Or do they have a "comeback" in one of the other PRs?

Reply:
At the moment they're unused. Let's try to clean up the current situation first, so that we can merge again (because we have a bugfix for a memory leak) and get into re-organising the graphql tests later.